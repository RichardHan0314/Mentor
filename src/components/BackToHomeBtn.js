import React from 'react';
import styled from 'styled-components';

const BackToHomeBtn = () => {
    return (
        <Button>Go back to home</Button>
    );
};

const Button = styled.button`
    height: 60px;
    padding: 10px 20px;
    background: #232323;
    font-size: 1.6rem;
    color: #f65261;
    outline: 1px solid #f65261;
    border: none;
    text-transform: uppercase;
`;

export default BackToHomeBtn;
