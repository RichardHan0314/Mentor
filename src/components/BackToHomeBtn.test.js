import React from 'react';
import { render, screen } from '@testing-library/react';
import BackToHomeBtn from './BackToHomeBtn';

describe('BackToHomeBtn component', () => {
    test('render BackToHomeBtn snapshot', () => {
        const { asFragment } = render(<BackToHomeBtn />);
        expect(asFragment(<BackToHomeBtn />)).toMatchSnapshot();
    });

    it('Render BackToHomeBtn correctly', () => {
        render(<BackToHomeBtn />);
        screen.debug();
        expect(screen.getByText(/Go back to home/)).toBeInTheDocument();
    });
});
