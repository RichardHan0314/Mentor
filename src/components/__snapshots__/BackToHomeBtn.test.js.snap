// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackToHomeBtn component render BackToHomeBtn snapshot 1`] = `
<DocumentFragment>
  <button
    class="sc-AxjAm dRfPaF"
  >
    Go back to home
  </button>
</DocumentFragment>
`;
